fluent-syntax/src/ast.js

```diff
@@ -194,27 +193,49 @@ export class Identifier extends SyntaxNode {
   }
 }
 
-export class Symbol extends Identifier {
+export class VariantName extends Identifier {
   constructor(name) {
     super(name);
-    this.type = 'Symbol';
+    this.type = 'VariantName';
   }
 }
 
-export class Comment extends Entry {
+export class BaseComment extends Entry {
   constructor(content) {
     super();
     this.type = 'Comment';
     this.content = content;
   }
 }
 
+export class Comment extends BaseComment {
+  constructor(content) {
+    super();
```

Comment:
You could pass `content` to `super` and avoid setting it in again in the constructor here.